instrumentation/opentelemetry-instrumentation-confluent-kafka/src/opentelemetry/instrumentation/confluent_kafka/__init__.py

```diff
@@ -178,9 +183,11 @@ def commit(self, *args, **kwargs):
         return self._consumer.commit(*args, **kwargs)
 
     def consume(
-        self, num_messages=1, *args, **kwargs
```

Comment:
Why is `num_messages` removed? Can't it be merged with `kwargs`?

Reply:
I removed `num_messages` because it wasn't used anywhere in the code. I think this function signature reflects better what the function is doing, passing everything down the line to the original function.

I can revert this change if you think it's better, no problem at all.